### PR TITLE
[DOCS] k0smotron edits for Status and Known Limitations section.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
k0smotron edits for Status and Known Limitations section.

### CODE REVIEW

File: docs/status.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
 # Project status
 
-We’re really early on with the project and as always with any young project there are probably some sharp corners. But with the help of the open source community we plan to iron out those in the coming months.
+To gauge the level of interest in running k0s control planes as Kubernetes
+resources, we are releasing k0smotron at the earliest possible moment. Of
+cousre, as with any young project just getting started, there are bound to be
```
Comment (makhov) [inline docs/status.md:5]: A typo in "cousre"

File: docs/status.md [hunk 2]
```diff
@@ -1,20 +1,41 @@
...
+control plane (within the management cluster) and the worker nodes in your
+favourite infrastructure supporting cluster API.
 
+## Known limitations
 
-## Cluster API
+Areas in which k0smotron is currently limited include:
 
-One of the directions we're looking at is the have k0smotron working as a Cluster API provider for both `ControlPlane` and worker `Bootstrap` providers. What this means is that you could utilize Cluster API to provision both the control plane, within the management cluster, and worker nodes in your favourite infrastructure supporting cluster API.
+* Control Plane configurability
```
Comment (makhov) [inline docs/status.md:28]: This section is outdated, now we support k0s configuration fully

File: docs/status.md [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
 
-## Known limitations
+    The configurability of the [k0s](https://docs.k0sproject.io/stable/) control plane is not yet fully enabled.
+
+* Control plane exposure
+
+    Currently k0smotron only supports `NodePort` and `LoadBalancer` services,
```
Comment (makhov) [inline docs/status.md:34]: Now it's `ClusterIP`, `NodePort`, and `LoadBalancer`. But everything else is true.

File: docs/status.md [hunk 4]
```diff
@@ -1,20 +1,41 @@
...
+    their interraction with cloud provider implementations.
+
+* Updates prevalence
 
-Some of the areas we know have lot of shortcomings currently:
-- Control Plane configurability: As you know, k0s itself has lot of configurability, we plan to enable full configurability of the k0s control plane
-- Control plane exposing: Currently k0smotron only supports `NodePort` and `LoadBalancer` type services. While that is in itself quite ok quite often there's need to further configure e.g. annotations etc. on the created service to make them play nice with cloud provider implementations.
-- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
-- 
+    Although k0smotron can easily update the cluster controlplane, such updates
```
Comment (makhov) [inline docs/status.md:40]: This is true for standalone mode

File: docs/status.md [hunk 5]
```diff
@@ -1,20 +1,41 @@
...
+Kubernetes deployments. We are confident that the open source community will
+take great interest in the project, too, and help us to smooth out the rough
+edges in the days, weeks, and months ahead.
 
-At this point we can not give much guarantees over backwards and forwards compatibility so expect things to break a bit with upcoming releases of k0smotron.
+## Cluster API
 
-We wanted to release k0smotron as early as possible to see whether there is in general interest of running k0s control planes as Kubernetes resources. We are excited about the potential that k0smotron brings to the table, and we look forward to seeing how it can transform the way you manage your Kubernetes deployments.
+One key focus of the k0smotron project is to get it to function as a Cluster
+API provider for both `ControlPlane` and worker `Bootstrap` providers. Once
```
Comment (makhov) [inline docs/status.md:19]: Currently, k0smotron can act as `ControlPlane`, `Bootstrap`, and `Infrastructure` provider. 
"(within the management cluster)" is redundant, with k0smotron currently we can provision the control plane both inside and outside the management cluster

File: docs/status.md (hunk 4 above)
Comment (KoryKessel-Mirantis) [reply docs/status.md:40]: Inserted reference to standalone mode into bullet.

File: docs/status.md (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply docs/status.md:28]: Removed section.

**Review by jnummelin — APPROVED** [review]

**Pull request merged**